src/models/Activity.ts

```diff
@@ -17,7 +17,7 @@ export interface IActivity {
     testerStaffId: string;
     testerEmail?: string;
     startTime?: string;
-    endTime?: string;
+    endTime?: null | string;
```

Comment:
Is null based on data retrieved from dynamo?

Reply:
endTime is null when the activity (we can have 3 types of activities; visit, wait time, unaccounted time) is still open and passed as default value when creating the visit from the FE. We write a date string when the activity is updated